### ISSUE TITLE
TCP Gecko never connects, even after installing and putting in right IP address

### ISSUE THREAD

**Issue by MKWPlayer1337** [BullyWiiPlaza/TCP-Gecko#1]
Do you think this is an issue? I went to loadiine, and ran the installer, worked fine, but I can't connect at all to the client for some odd reason. Any help would be appreciated. thanks

**Comment by BullyWiiPlaza** [comment]
Does it work with the other TCP Gecko Installers? I doubt it so it's an issue on your end.

**Comment by MKWPlayer1337** [comment]
what do you mean by other TCP Gecko installers? I tried the one thru the homebrew menu, loadiine, and the wj44.ml process. All of them result in the same error. Oddly J Gecko U connects just fine.

**Comment by BullyWiiPlaza** [comment]
You mean TCP Gecko.NET? Some versions have an error popup but it can be ignored.

**Comment by MKWPlayer1337** [comment]
Yeah... I am using the TCP Gecko .NET from your installation tutorial on YouTube. (btw this is Matt224) I created this account many years ago so the name was different than what I go by now. But yea. Same problems unfortunately. :/ Also I can see what you mean as this being a problem on my end.
